kic: add events section to custom entities guide (#8246)
Kong/docs.konghq.com: app/_src/kubernetes-ingress-controller/guides/services/custom-entity.md

## Create a GraphQL Service

> The `degraphql` plugin requires an upstream GraphQL API. For this tutorial, we'll use [hasura] to create an example GraphQL service:
> 
> [hasura]: https://hasura.io/

Reviewer: ```suggestion
The `degraphql` plugin requires an upstream GraphQL API. For this tutorial, we'll use [Hasura] to create an example GraphQL service:

[Hasura]: https://hasura.io/
```
Why switch this to lowercase? Based on a quick google, their product name is "Hasura" with a capital.

Author: Good call!

which matches the data inserted in the previous steps.

{% if_version gte:3.4.x %}

> ## Invalid `KongCustomEntity` Configuration

Reviewer: ```suggestion
## Troubleshooting
### Invalid `KongCustomEntity` Configuration
```
Let's nest it under a troubleshooting section, otherwise the heading/section name feels out of place.

Each `KongCustomEntity` is validated against the schema from Kong.

> If the configuration is invalid, an `Event` with reason set to `KongConfigurationTranslationFailed` will be emitted.

Reviewer: ```suggestion
If the configuration is invalid, an `Event` with the reason set to `KongConfigurationTranslationFailed` will be emitted.
```

The `involvedObject` of this `Event` will be set to the `KongCustomEntity` resource.

> For more information on observability with events please consult our [events guide][events_guide].

Reviewer: ```suggestion
For more information on observability with events, see our [events guide][events_guide].
```
